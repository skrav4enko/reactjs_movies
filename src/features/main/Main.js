import React from 'react';
import './Main.scss';
import Header from '../../core/header/Header';
import Footer from '../../core/footer/Footer';
import SearchMovie from '../search-movie/SearchMovie';
import MovieDetail from '../movie-detail/MovieDetail';

import moviesData from '../../mock/data.json';

const movies = [...moviesData.data];

function Main(props) {
  return (
    <div className="main">
      <Header />
      <main className="content">
        <SearchMovie movies={movies} />
        {/* <MovieDetail movie={movies[0]} movies={movies} /> */}
      </main>
      <Footer />
    </div>
  );
}

export default Main;
