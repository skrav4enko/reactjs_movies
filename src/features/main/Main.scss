.main {
  position: relative;
  background-color: #232323;
  color: #ffffff;
  height: 100%;
  display: grid;
  grid-template-rows: 1fr 70px;

  .content {
    display: flex;
    flex-direction: column;
  }
}
