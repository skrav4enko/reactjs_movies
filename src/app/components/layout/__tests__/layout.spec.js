/* eslint-disable */
import React from 'react';
import { shallow } from 'enzyme';
import Main from '..';

describe('Layout', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Main />);
    expect(wrapper).toMatchSnapshot();
  });
});

