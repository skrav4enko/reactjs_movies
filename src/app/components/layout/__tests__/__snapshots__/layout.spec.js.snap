// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should match snapshot 1`] = `<index.tsx />`;
